FirstTask.py

```diff
@@ -0,0 +1,230 @@
+            return False, f"{account_not_found} Account Not Found!"
+    
+    def compare(self, current_account, other_account):
+        status_1, response_1 = self.find_account(current_account)
+        status_2, response_2 = self.find_account(other_account)
+        if status_1 and status_2:
+            if response_1.check_balance() > response_2.check_balance():
+                return True, "Your Account is Greater than Provided Account!"
+            if response_1.check_balance() < response_2.check_balance():
+                return True, "Your Account is Smaller than Provided Account!"
+            if response_1.check_balance() == response_2.check_balance():
+                return True, "Your Account is Equal to the Provided Account!"
+        else:
+            if status_1 == False:
+                account_not_found = current_account
+            elif status_2 == False:
+                account_not_found = other_account
+            if status_1 == False and status_2 == False:
+                return False, f"{current_account} and {other_account} are invalid!"
+            return False, f"{account_not_found} Account Not Found!"
+   
+    def account_details(self, account_number):
+        status,response = self.find_account(account_number)
+        if status:
+            response.account_details()
+        else:
+            return False, f"{account_number} Account Not Found!"
+   
+    def logout(self):
+        self.logged_in_account = None
+        return self.logged_in_account
+
+    def is_logged_in(self):
+        if self.logged_in_account:
+            return True
+        return False
+
+def run():
+    bank = Bank()
+    while(True):
+        option = int(input("1) Create Account \n2) Login \n3) Find Account \n4) Total Accounts\n0) Exit \nChoose:"))
+        if option == 1:
+            sys('clear')
+            name = input("Enter Full Name: ")
+            pass_code = int(input("Enter Passcode "))
+            while(True):
+                msg = "\n".join(f'{index + 1}) {account_type}' for index, account_type in enumerate(bank.ACCOUNT_TYPES))
+                print(msg)
+                account_select = int(input())
+                if 0 < account_select <= len(bank.ACCOUNT_TYPES):
+                    account_type = bank.ACCOUNT_TYPES[account_select - 1]   
+                    break 
+                else:
+                    print('\Invalid Selection\n')
+                input('Press any key to continue!')
+            print("Account Created and your ID is:", bank.create_account(name, pass_code, account_type))
+            input("\nPress Any Key to Continue")
+            sys("clear")
+        
+        elif option == 2:
+            if bank.is_logged_in() == False:
+                account_number = int(input("Enter Account Number: "))
+                code = int(input("Enter Passcode: "))
+                status, response = bank.login(account_number, code)
+
+            if bank.is_logged_in():

```

Comment:
We should use the returned `status` instead of calling another method to do the same.
```suggestion
            if status:
```

Reply:
Also, we are not handling the `else` part of this. Meaning if the user is not logged in we need to show the message and go back to the previous menu.